pymc4/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+"""
+PyMC4 test configuration
+"""
+import pytest
+import tensorflow as tf
+
+
+@pytest.fixture(scope="function")
+def tf_session():
+    sess = tf.Session()
```

Comment:
How about:
```
with tf.Graph().as_default():
    with tf.Session() as sess:
        yield sess
```

Reply:
Do we need to open and close session each time? I figured one session was fine if we cleared the graph, and opening and closing them would be a waste of computational resources, but I could be wrong

If we do implement this I'd prefer `sess.close()` in the finalizer since it'll make it easier to avoid stacked context managers, which then makes debugging easier and mocking easier imo

Reply:
I'd vote for the current solution with graph reset. There is no difference between them. I see no computational resources waste...

Reply:
@ferrine this suggests they are different: https://stackoverflow.com/questions/42706761/closing-session-in-tensorflow-doesnt-reset-graph

Reply:
I think it's quite easy to run into memory problems here if we keep adding to graph or create multiple sessions in these tests.